src/minoversikt/huskelapp/redigering/slettEksisterendeArbeidsliste.ts

```diff
@@ -1,31 +1,22 @@
-import {ArbeidslisteDataModell} from '../../../model-interfaces';
+import {BrukerModell} from '../../../model-interfaces';
 import {visServerfeilModal} from '../../../ducks/modal-serverfeil';
-import {FJERN_FRA_ARBEIDSLISTE_FEILET, visFeiletModal} from '../../../ducks/modal-feilmelding-brukere';
 import {leggTilStatustall} from '../../../ducks/statustall-veileder';
 import {oppdaterArbeidslisteForBruker} from '../../../ducks/portefolje';
+import {ThunkDispatch} from 'redux-thunk';
+import {AppState} from '../../../reducer';
+import {AnyAction} from 'redux';
+import {slettArbeidslisteUtenFargekategori} from '../../../middleware/api';
 
-export const oppdaterStateVedSlettArbeidsliste = (res, arbeidsliste: ArbeidslisteDataModell[], dispatch) => {
-    if (!res) {
+export const slettArbeidslisteUtenFargekategoriOgOppdaterRedux = async (
```

Comment:
Kanskje vi kunne kalla den "slettArbeidslisteMenIkkeFargekategoriOgOppdaterRedux", det høyrest litt ut som om arbeidslista ikkje _har_ fargekategori i staden for at den har og vi ikkje vil slette kategorien. Gjeld også på linje 15 i denne fila